### DISCUSSION TITLE
Row and column coloring [Jul 22]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#2880] [Discord Threads]
> 🙋‍♂️ **Kelsey**

**Comment by o1lab** [comment]
> 👨‍🦲 **dstala**  
#### <@999707539128197202> not supported  
#  
> 🙋‍♂️ **Kelsey**  
#### thank you. Maybe something to look into adding on next version  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/999709896331231232)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
